packages/abstract-boot/src/deployment.rs

```diff
@@ -58,102 +56,105 @@ impl<Chain: BootEnvironment> boot_core::Deploy<Chain> for Abstract<Chain> {
                     ::module_factory::contract::instantiate,
                     ::module_factory::contract::query,
                 )
-                .with_reply_empty(::module_factory::contract::reply),
+                .with_reply_empty(::module_factory::contract::reply)
+                .with_migrate(::module_factory::contract::migrate),
             ));
 
             version_control.as_instance_mut().set_mock(Box::new(
                 boot_core::ContractWrapper::new_with_empty(
                     ::version_control::contract::execute,
                     ::version_control::contract::instantiate,
                     ::version_control::contract::query,
-                ),
+                )
+                .with_migrate(::version_control::contract::migrate),
             ));
 
             manager.as_instance_mut().set_mock(Box::new(
                 boot_core::ContractWrapper::new_with_empty(
                     ::manager::contract::execute,
                     ::manager::contract::instantiate,
                     ::manager::contract::query,
-                ),
+                )
+                .with_migrate(::manager::contract::migrate),
             ));
 
-            proxy
-                .as_instance_mut()
-                .set_mock(Box::new(boot_core::ContractWrapper::new_with_empty(
+            proxy.as_instance_mut().set_mock(Box::new(
+                boot_core::ContractWrapper::new_with_empty(
                     ::proxy::contract::execute,
                     ::proxy::contract::instantiate,
                     ::proxy::contract::query,
-                )));
+                )
+                .with_migrate(::proxy::contract::migrate),
+            ));
         }
 
-        let mut deployment = Abstract {
-            chain,
-            version,
```

Comment:
Assuming that version was removed because it's pulled from the env?

Reply:
And there's no good reason for people to have 2 different versions of Abstract deployed...